<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HydroDownloaderDlg</class>
 <widget class="QDialog" name="HydroDownloaderDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>875</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>SelectStation</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="SelectStationcomboBox"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Select State</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="StatescomboBox"/>
     </item>
     <item row="2" column="1">
      <widget class="QDateEdit" name="dateEditStart">
       <property name="displayFormat">
        <string>MM/dd/yyyy</string>
       </property>
       <property name="date">
        <date>
         <year>2010</year>
         <month>1</month>
         <day>1</day>
        </date>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Start Date</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>End Date</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDateEdit" name="dateEditEnd">
       <property name="displayFormat">
        <string>MM/dd/yyyy</string>
       </property>
       <property name="date">
        <date>
         <year>2010</year>
         <month>1</month>
         <day>1</day>
        </date>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="RetrieveDataBtn">
         <property name="text">
          <string>Retrieve Data</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ExporttoCSV">
         <property name="text">
          <string>Export to CSV</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
